homeworks/Vladimir.Karbivnychiy_VladimirSkarb/13-ng-forms/login-form/src/app/components/login-form/login-form.component.ts

```diff
@@ -0,0 +1,54 @@
+import {Component, OnInit} from '@angular/core';
+import {
+  FormGroup,
+  FormControl,
+  Validators
+} from '@angular/forms';
+
+import {User} from "../../models/user.model";
+
+@Component({
+  selector: 'app-login-form',
+  templateUrl: './login-form.component.html',
+  styleUrls: ['./login-form.component.css']
+})
+export class LoginFormComponent implements OnInit {
+
+  ngOnInit() {
+    this.checkStorage();
+  }
+
+  loginForm = new FormGroup({
+    email: new FormControl('', [
+      Validators.required,
+      Validators.email
+    ]),
+    password: new FormControl('', [
+      Validators.minLength(8)
+    ]),
+    checkbox: new FormControl(),
+  });
+
+
+  onSubmit(): void {
+    const info: string = `Email - ${this.loginForm.value.email}, Password - ${this.loginForm.value.password}`
+    if (this.loginForm.value.checkbox) {
```

Comment:
But what if I once pressed the checkbox and then leaved it disabled? Each time I reload the page the saved user will be loaded to the form :smile: 

So please, add additional logic to clear the user if checkbox is NOT selected :slightly_smiling_face: